README.md

```diff
@@ -69,6 +69,7 @@ The complete list of environment variable configuration options is:
 1. `BUSTY_ATTACHMENT_DIR` - Directory to save attachments (default = attachments)
 1. `BUSTY_DJ_ROLE` - Name of role with permissions to run commands (default = bangermeister)
 1. `BUSTY_CUSTOM_EMOJI_FILEPATH` - The Python module to import containing the emoji list (default = emoji_list)
+1. `BUSTY_IMAGE_STATE_FILE` - The location of the file which saves the state of the currently loaded image (default = .image_state)
```

Comment:
Starting to wonder if we should just have a "data" folder that this file and an "attachments" subfolder could live in, rather than have an env var for each. What do you think?

Reply:
I like this idea, I say we do it in a separate commit after this one to keep commits small and single-purpose